bruno: 2.8.1 -> 2.9.1

water-sucks opened the pull request:
Updates `bruno` to 2.9.1. The automatic update PR crashes since a new package, `@usebruno/filestore`, was not being built in the derivation.

Changelog: https://github.com/usebruno/bruno/compare/v2.8.1...v2.9.1

Supersedes #433573.

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform:
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [ ] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [ ] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [x] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [x] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

water-sucks committed:
bruno: 2.8.1 -> 2.9.1

gepbird reviewed (changes requested):
Thanks for picking this up!

Can you also do the same change for `bruno-cli` package? Since it depends on `bruno`, it has the same issue

water-sucks commented:
I added it to the same commit @gepbird, since it felt weird to separate a change like this between two packages that depended on the same source.

Seems to launch the CLI fine and run basic requests after I added it.

ddogfoodd reviewed (approved):
works for me as well, thank you!

gepbird reviewed (approved):
Tested the binaries on x86_64-linux, looks good!

I also think it's better if you put these changes in a single commit. Maybe you can indicate that in the commit message and PR title like `bruno{,-cli}: 2.8.1 -> 2.9.1`

MattSturgeon reviewed (approved):
Thanks!
